src/dcp/dcp.c

```diff
@@ -491,14 +497,14 @@ int main(int argc, char** argv)
     /* connect to DAOS source pool if uuid is valid */
     if (mfu_src_file->type == DAOS) {
         /* if DAOS source pool uuid is valid, then set source file type to DAOS */
-        daos_connect(rank, svc, src_pool_uuid, src_cont_uuid, &src_poh, &src_coh); 
+        daos_connect(rank, src_svc, src_pool_uuid, src_cont_uuid, &src_poh, &src_coh); 
     }
 
     if (mfu_dst_file->type == DAOS) {
         if (daos_uuid_valid(dst_pool_uuid) && !same_pool) {
             /* if DAOS is the source and destination type, and containers are in different pools,
              * then connect to the second pool */
-            daos_connect(rank, svc, dst_pool_uuid, dst_cont_uuid, &dst_poh, &dst_coh); 
+            daos_connect(rank, dst_svc, dst_pool_uuid, dst_cont_uuid, &dst_poh, &dst_coh); 
```

Comment:
@adammoody that is a good question. It isn't totally clear to me the more I think about it, because I'm not sure if you would just connect to the pool with the greater svc or connect to the same pool twice with different service levels. I will ask about the best way to do this.

Reply:
> I don't know the details well enough, but I see this is an input arg to connect. Would we also need to call connect twice if dst_svc != src_svc?

@adammoody I updated this to print an error in the case that the same pool uuid is being used and the dst_svc and src_svc are not equal. This would likely never happen and if it does it is really not efficient to connect to the same pool twice when you can reuse the handle, so I am logging an error here.

Reply:
Thanks, the spacing looks off in the code you just added, maybe some TABS work their way in?  Other than that, it's good with me.